### PR TITLE
Add Instant Access Plugin

### PR DESCRIPTION
**Pull request by erguotou520**
Request to add Instant Access notification plugin.

### CODE REVIEW

File: content/erguotou/drone-instant-access.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+date: 2020-03-07T00:00:00+08:00
+title: InstantAccess(即时达)
+author: erguotou
```
Comment (tboerger) [inline content/erguotou/drone-instant-access.md]: The author is so far matching the GitHub handle, so this should be `erguotou520 `

File: content/erguotou/drone-instant-access.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+tags: [ notifications, InstantAccess, 即时达 ]
+logo: instant_access.svg
+repo: erguotou520/drone-instant-access
+image: erguotou/drone-instant-access
+---
+
+The InstantAccess plugin posts build status messages to your account. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+---
```
Comment (tboerger) [inline content/erguotou/drone-instant-access.md]: For all the other examples we are not using a full pipeline, just the steps required for the plugin. Please use the other plugins as an example.

Comment (erguotou520) [reply content/erguotou/drone-instant-access.md]: So just remove
```
---
kind: pipeline
name: default
```
?

File: content/erguotou/drone-instant-access.md (hunk 1 above)
Comment (erguotou520) [reply content/erguotou/drone-instant-access.md]: fixed

File: content/erguotou/drone-instant-access.md (hunk 2 above)
Comment (erguotou520) [reply content/erguotou/drone-instant-access.md]: Doc updated

Comment (tboerger) [reply content/erguotou/drone-instant-access.md]: Exactly like you have done it now.

**Pull request merged**